src/MetalPerformanceShaders/MPSDefs.cs

```diff
@@ -81,10 +83,19 @@ public struct MPSScaleTransform {
 	}
 
 	// MPSImageHistogram.h
+	[StructLayout (LayoutKind.Explicit)]
 	public struct MPSImageHistogramInfo {
+		[FieldOffset (0)]
 		public nuint NumberOfHistogramEntries;
+#if XAMCORE_2_0
+		[FieldOffset (8)]
+#else
+		[FieldOffset (4)]
+#endif
```

Comment:
What's the story here ? :-)
Only XM classic won't defined `XAMCORE_2_0` and IIRC it does not support Metal, so it looks like _dead_ code.